cmd/arc_summary/arc_summary3

```diff
@@ -788,8 +837,8 @@ def section_spl(*_):
     and/or descriptions. This does not use kstats.
     """
 
-    if sys.platform.startswith('freebsd'):
-        # No SPL support in FreeBSD
+    if sys.platform.startswith('freebsd') or sys.platform.startswith("darwin"):
+        # No SPL support in FreeBSD or OS X
```

Comment:
We do have SPL and a lot of those values (almost identical to illumos) but not entirely sure what they mean by spl params or which ones. Probably easier to skip for now, and come back to it again if we feel like it

Reply:
I could look at pulling out kstat.spl and see if that makes it show up anything useful.

Reply:
Ah looks like it sort of just dumps them, doesn't really do anything useful with them. We certainly could as well, but not sure that's any different to running sysctl.  Similarly, tunables?